pyiron/vasp/base.py

```diff
@@ -466,20 +466,54 @@ def collect_errors(self):
         """
         Collects errors from the VASP run
         """
-        num_eddrmm, snap = self._get_eddrmm_info()
 
-        warning_string = "EDDRMM warnings occured {} times, first in ionic step {}."
-        status_string = "Status is switched to 'warning'."
+        # error messages by VASP
+        eddrmm_error_str = "WARNING in EDDRMM: call to ZHEGV failed, returncode ="
+        zbrent_error_str = "ZBRENT: fatal error in bracketing"
 
-        if snap is not None:
+        # warning messages for pyiron
+        eddrmm_warning_str = "EDDRMM warnings occured {} times, first in ionic step {}."
+        zbrent_warning_str = "'ZBRENT: fatal error in bracketing' occured. Please check VASP manual for details."
+        warning_status_str = "Status is switched to 'warning'."
+        aborted_status_str = "Status is switched to 'aborted'."
+
+        # collecting errors
+        num_eddrmm = 0
+        snap_eddrmm = None
+
+        zbrent_status = False
+
+        file_name = os.path.join(self.working_directory, "error.out")
+        if os.path.exists(file_name):
+            with open(file_name, "r") as f:
+                lines = f.readlines()
+
+            # EDDRMM
+            # If the wrong convergence algorithm is chosen, we get the following error.
+            # https://cms.mpi.univie.ac.at/vasp-forum/viewtopic.php?f=4&t=17071
+            lines_where_eddrmm = np.argwhere([eddrmm_error_str in l for l in lines]).flatten()
+            num_eddrmm = len(lines_where_eddrmm)
+            if num_eddrmm > 0:
+                snap_eddrmm = len(np.argwhere(["E0=" in l for l in lines[:lines_where_eddrmm[0]]]).flatten())
+
+            # ZBRENT
+            for l in lines:
+                if zbrent_error_str in l:
+                    zbrent_status = True
```

Comment:
We could add a `break` statement here which means it will stop the loop once `ZBRENT` is seen.

Reply:
Fair enough. Done